### PR TITLE
Directories added to dirChan no longer trigger checkDir

### PR DESCRIPTION
**Pull request by Michael-Gallo**
In certain conditions, `checkDir` would cause a directory to be sent to `app.nav.dirChan` which would in turn call `checkDir`, creating an infinite loop.  The call to `checkDir` when a value is sent to `app.nav.dirChan` is likely unnecessary and removing it should fix the loop
See Issue #1543

### CODE REVIEW

**Comment by joelim-work** [comment]
Thanks, I would also consider removing the following two lines of code since they were also introduced in the same commit, and is also a potential source for infinite loops: https://github.com/gokcehan/lf/blob/cf996263bb80b7da8fffbea81eba5fa2c0d02378/app.go#L419-L420

**Comment by gokcehan** [comment]
@joelim-work @Michael-Gallo Is this waiting for my approval? Sorry, if that's the case. I think it looks fine and we can merge it.

**Review by gokcehan — APPROVED** [review]

**Pull request merged**
